src/terminal.zig

```diff
@@ -86,27 +87,14 @@ pub fn cursorPosition(stdout: fs.File, stdin: fs.File) !Pos {
 }
 
 fn enableRawMode(file: fs.File) !void {
-    var raw = try getAttr(file);
-    raw.c_iflag &= ~@as(@TypeOf(raw.c_lflag), BRKINT | ICRNL | INPCK | ISTRIP | IXON);
-    raw.c_oflag &= ~@as(@TypeOf(raw.c_lflag), c.OPOST);
-    raw.c_cflag &= ~@as(@TypeOf(raw.c_lflag), c.CS8);
-    raw.c_lflag &= ~@as(@TypeOf(raw.c_lflag), c.ECHO | c.ICANON | c.IEXTEN | c.ISIG);
-    raw.c_cc[c.VMIN] = 0;
-    raw.c_cc[c.VTIME] = 1;
-    try setAttr(file, raw);
-}
-
-fn getAttr(file: fs.File) !c.termios {
-    var raw: c.termios = undefined;
-    if (c.tcgetattr(file.handle, &raw) == -1)
-        return error.TermiosError;
-
-    return raw;
-}
-
-fn setAttr(file: fs.File, attr: c.termios) !void {
-    if (c.tcsetattr(file.handle, c.TCSAFLUSH, &attr) == -1)
-        return error.TermiosError;
+    var raw = try os.tcgetattr(file.handle);
+    raw.iflag &= ~@as(@TypeOf(raw.lflag), BRKINT | ICRNL | INPCK | ISTRIP | IXON);
+    raw.oflag &= ~@as(@TypeOf(raw.lflag), os.OPOST);
+    raw.cflag &= ~@as(@TypeOf(raw.lflag), os.CS8);
+    raw.lflag &= ~@as(@TypeOf(raw.lflag), os.ECHO | os.ICANON | os.IEXTEN | os.ISIG);
+    raw.cc[VMIN] = 0;
+    raw.cc[VTIME] = 1;
```

Comment:
Hmm. I don't see these define in this file. Does this even compile?

Reply:
You defined them in one of your very first commits. They are here: https://github.com/Hejsil/zte/blob/master/src/terminal.zig#L117

Reply:
:thinking: I guess i did!